requirements/default.txt

```diff
@@ -209,6 +209,9 @@ MySQL-python==1.2.3c1
 # sha256: sboSwcRTI_2Glb5oznIoRWuG5RI08WcCfrvEzNqFLGE
 nose==1.1.2
 
+# sha256: REZqm8tW0uVodQ-RUE0SeMdOq7JZowWwbpdbh7UWNdo
+nose-exclude==0.4.1
```

Comment:
I see now why this can't go in `test.txt`, since it is used to exclude tests during normal test runs. It should go in `dev.txt`.